src/Raven.Server/Documents/PeriodicBackup/Aws/RavenAwsGlacierClient.cs

```diff
@@ -171,17 +174,21 @@ private async Task UploadPart(Stream baseStream, HttpClient client, string url,
 
             // wait for one second before trying again to send the request
             // maybe there was a network issue?
-            await Task.Delay(1000);
+            Thread.Sleep(1000);
```

Comment:
Don't we need to take advantage of the cancellation token here? For the server / db shutdown?

Reply:
done